docs/reference/list_field_definition.rst

```diff
@@ -1,182 +0,0 @@
-* string
-* text
-* date
-* time
-* array
-
-If no type is set, the ``Admin`` class will use the type defined in the doctrine
-mapping definition.
-
-List Actions
-------------
-
-You can set actions for the list items by adding an '_action' field in ``configureListFields``::
-
-    $listMapper
-        ->add('_action', 'actions', [
-            'actions' => [
-                'view' => [],
-                'edit' => [],
-            ]
-        ]);
-
-Edit and delete actions are enabled in the default configuration. You can add
-your own! Default template file is: ``@SonataAdmin/CRUD/list__action_[ACTION_NAME].html.twig``
-
-You can specify your own by setting up the 'template' option like so::
-
-    $listMapper
-        ->add('_action', 'actions', [
-            'actions' => [
-                'view' => [],
-                'edit' => [],
-                'delete' => ['template' => '@My/MyController/my_partial.html.twig'],
-            ]
-        ]);
-
-Advance Usage
--------------
-
-Displaying sub entity properties
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-If you need to display only one field from a sub entity in a dedicated column,
-you can use the dot-separated notation (note that this only makes sense when
-the prefix path is made of entities, not collections)::
-
-    namespace App\Admin;
-
-    use Sonata\AdminBundle\Admin\AbstractAdmin;
-    use Sonata\AdminBundle\Datagrid\ListMapper;
-
-    final class UserAdmin extends AbstractAdmin
-    {
-        protected function configureListFields(ListMapper $listMapper)
-        {
-            $listMapper
-                ->addIdentifier('id')
-                ->add('firstName')
-                ->add('lastName')
-                ->add('address.street')
-                ->add('address.ZIPCode')
-                ->add('address.town')
-            ;
-        }
-    }
-
-Custom template
-^^^^^^^^^^^^^^^
-
-If you need a specific layout for a row cell, you can define a custom template::
-
-    namespace Sonata\MediaBundle\Admin;
-
-    use Sonata\AdminBundle\Admin\AbstractAdmin;
-    use Sonata\AdminBundle\Datagrid\ListMapper;
-
-    final class MediaAdmin extends AbstractAdmin
-    {
-        protected function configureListFields(ListMapper $listMapper)
-        {
-            $listMapper
-                ->addIdentifier('id')
-                ->add('image', 'string', [
-                    'template' => '@SonataMedia/MediaAdmin/list_image.html.twig',
-                ])
-                ->add('custom', 'string', [
-                    'template' => '@SonataMedia/MediaAdmin/list_custom.html.twig',
-                ])
-            ;
-        }
-    }
-
-The related template :
-
-.. code-block:: jinja
-
-    {% extends '@SonataAdmin/CRUD/base_list_field.html.twig' %}
-
-    {% block field%}
-        <div>
-            <strong>{{ object.name }}</strong> <br/>
-            {{ object.providername}} : {{ object.width }}x{{ object.height }} <br/>
-        </div>
-    {% endblock %}
-
-Custom link
```

Comment:
I'm not sure this part is in SonataAdmin.
If not, it should be moved (because it's not related to the persistence bundle).
I'll check later, I'm currently on phone.

Reply:
I thought it was copy past from DoctrineBundle

Reply:
Seems like it was added separately.

But I look at the AdminBundle doc, and it's already in the ```FieldDescriptionInterface::TYPE_URL``` section